CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -344,17 +357,17 @@ class CameraFragment : Fragment() {
      *  @param height - preview height
      *  @return suitable aspect ratio
      */
-    private fun aspectRatio(width: Int, height: Int): AspectRatio {
+    private fun aspectRatio(width: Int, height: Int): Int {
```

Comment:
Why are you removing the declared typing information?

Reply:
It's the wrong return type; eg. `AspectRatio.RATIO_4_3` is `0` and `AspectRatio.RATIO_16_9` is `1`.